scripts/script_utils.py

```diff
@@ -136,6 +136,19 @@ def load_rules_from_cache(args) -> List[dict]:
         with open(rules_file, "rb") as f:
             rules_data = pickle.load(f)
             rules = [rules_data.get(key) for key in keys]
+
+        missing_rules = [rule for rule, data in zip(args.rules, rules) if data is None]
+        if missing_rules:
+            for missing_rule in missing_rules:
+                engine_logger.error(
+                    f"The rule specified '{missing_rule}' is not"
+                    " in the standard {args.standard} and version {args.version}"
+                )
+        rules = [rule for rule in rules if rule is not None]
+        if not rules:
+            raise ValueError(
+                "All specified rules were excluded because they are not in the standard and version specified."
+            )
     else:
         engine_logger.warning(
```

Comment:
I believe not specifying a rule and running all rules in the directory also should not be a warning.  I want to be consistent with the level of log.  I was going to change it to info.  Would you agree with the downgrade?